collection/rancher/v2.x/rancher-pod-collector/rancher-pod-collector.sh

```diff
@@ -0,0 +1,217 @@
+#!/bin/bash
+SPACE="512"
+TIMEOUT="60"
```

Comment:
What is this for?

Reply:
The variable `SPACE` is for the minimum free space in MBs for the tmp dir.
I removed the `TIMEOUT` that was leftover from the log script.